bundler.d/assets.rb

```diff
@@ -4,6 +4,6 @@
 gem 'gridster-rails', '~> 0.5'
 gem 'gettext_i18n_rails_js', '~> 1.0'
 gem 'execjs', '>= 1.4.0', '< 3.0'
-gem 'uglifier', '>= 1.0.3'
+gem 'uglifier', '>= 4.0.0'
```

Comment:
Our 5.1 SCL contains 3.2.0 so that'd need a bump.

Reply:
https://github.com/theforeman/tfm-ror51-packaging/pull/44